src/pybind/mgr/volumes/module.py

```diff
@@ -656,7 +658,9 @@ def handle_command(self, inbuf, cmd):
     def _cmd_fs_volume_create(self, inbuf, cmd):
         vol_id = cmd['name']
         placement = cmd.get('placement', '')
-        return self.vc.create_fs_volume(vol_id, placement)
+        data_pool = cmd.get('data_pool', None)
+        meta_pool = cmd.get('meta_pool', None)
```

Comment:
nit: extra new line above.

Reply:
weird - this showed up a _bit_ differently per-commit review :/

Reply:
Okay. Marking this as resolved since there is no extra newline character.